docs/pages/guides/schema-validation.mdx

```diff
@@ -48,10 +48,26 @@ the storage that do not match the schema you provided. Situations like this
 should only happen in development mode, and the developer is responsible for
 fixing them.
 
-## Create a schema
+## Schema Operations
+
+You can interact with the schemas you create through the dashboard or by calling
+the [REST API](/docs/api-reference/rest-api-endpoints). In this guide, we will
+cover the following operations:
+
+- [Create a schema](#Create-a-schema)
+- [Delete a schema](#Delete-a-schema)
+- [Attach a schema to a room](#Attach-a-schema-to-a-room)
+- [Detach a schema from a room](#Detach-a-schema-from-a-room)
```

Comment:
More of a style/consistency thing, not super important. But in other guides, our section titles are in present progressive tense form, e.g. "Creat**ing** a schema", "Detach**ing** a schema from a room" etc. Perhaps keep those consistent? No strong opinion here. Perhaps we don't do it consistently yet, in which case, feel free to disregard.